english/light-fire/magic-world/follow/dream/fight/sword/sword.md

```diff
@@ -5,3 +5,11 @@ You try to fight, but you failed fencing in high school.
 You're dead. 
 
 The rat smirks as you sink into the unknown. 
+
+
+
+... But wait! There is a light in the darkness. It seems to be growing in intensity.
+
+[Go into the light](https://github.com/udacity/create-your-own-adventure/blob/master/language.md)
```

Comment:
This link seems to be an absolute link to GitHub rather than a relative link.  If you are linking to another page in the create-your-own-adventure repository, please use relative links to do so.  See the README of this repository for more information.